data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
It's not about what the player would see from the screen. It was seen in the translation process by translators.